packages/xrpl/src/client/connection.ts

```diff
@@ -175,8 +176,10 @@ async function websocketSendAsync(
 export class Connection extends EventEmitter {
   private readonly url: string | undefined
   private ws: WebSocket | null = null
-  private reconnectTimeoutID: null | NodeJS.Timeout = null
-  private heartbeatIntervalID: null | NodeJS.Timeout = null
+  // Typing necessary for Jest tests running in browser
+  private reconnectTimeoutID: null | ReturnType<typeof setTimeout> = null
```

Comment:
What's the difference between this and what was previously there?

Reply:
It will use the value unique to the browser or nodejs, depending on the platform. The browser `setTimeout` does not have the same type as the nodejs one.